src/helper-modules/companies-house/wrap-companies-house-http-error-callback.ts

```diff
@@ -9,21 +9,25 @@ type CompaniesHouseHttpErrorCallback = (error: Error) => ObservableInput<never>;
 export const createWrapCompaniesHouseHttpGetErrorCallback =
   ({ messageForUnknownError, knownErrors }: { messageForUnknownError: string; knownErrors: KnownErrors }): CompaniesHouseHttpErrorCallback =>
   (error: Error) => {
-    if (error instanceof AxiosError && error.response && typeof error.response.data === 'object') {
-      const errorResponseData = error.response.data;
-      let errorMessage: string;
+    if (error instanceof AxiosError) {
+      let message: string;
```

Comment:
`string | undefined` as we've got an `if else` on lines `16-19` that might not fire if neither of the conditions are true

Reply:
Have removed as this might not be necessary anymore.